codesign/inputparse.go

```diff
@@ -0,0 +1,111 @@
+package codesign
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/bitrise-io/go-steputils/stepconf"
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/sliceutil"
+	"github.com/bitrise-io/go-xcode/autocodesign"
+	"github.com/bitrise-io/go-xcode/autocodesign/certdownloader"
+	"github.com/bitrise-io/go-xcode/autocodesign/keychain"
+)
+
+// StepInputParser ...
+type StepInputParser struct {
```

Comment:
Why does StepInputParser store the inputs, while StepInputParser.Parse does not have any parameter?

I would move code signing related Step Inputs into a separate struct (codesign.Inputs) and call the Parse return value as codesign.Config